json.go

```diff
@@ -0,0 +1,67 @@
+package ff
```

Comment:
Honest question: do you think it's better to move this to its own ffjson subpackage? Pro: symmetry with potential future ffyaml subpackage. Con: this one only uses stdlib, so not strictly necessary?

Reply:
Might make more sense.